fetch.js

```diff
@@ -1,23 +1,29 @@
-(function(self) {
+(function (root, factory) {
+  /* jshint strict:false */
+  /* globals define, module */
```

Comment:
I initially tried to use`/* jshint ignore: start */` (and end) but it didn't work in 2.8.0 jshint (maybe https://github.com/jshint/jshint/issues/2411). After upgrading to jshint 2.9.0 it works, but then it says factory is unused, and everything I tried to ignore that caused even more issues, and 2.9.0 also detects other undefined variables like Promise/Symbol. So this seemed to be the solution with the least amount of changes.